chamber/models/__init__.py

```diff
@@ -355,14 +355,29 @@ def _persistence_clean(self, *args, **kwargs):
     def _get_save_extra_kwargs(self):
         return {}
 
-    def _pre_save(self, *args, **kwargs):
+    def _pre_save(self, change, changed_fields, *args, **kwargs):
+        """
+        :param change: True if model instance was changed, False if was created
```

Comment:
Why is the argument called "change"? Wouldn't is_changed or changed make more sense?

Reply:
for future it will be better add both is_adding and is_changing value. But I left it for future pr